docs/drafts/developer.md

```diff
@@ -173,23 +154,39 @@ v0.33.1, built 2023-06-28
 
 Typically, you'll want to press the space bar to have it open the UI in your web browser.
 
-Shortly after starting, Tilt processes the `Tiltfile`, resulting in:
+---
+
+## Special Setup for MacOS
+
+Some additional setup is necessary on Macintosh computers to install and configure compatible tooling.
+
+### Install Homebrew and tools
+Follow the instructions to [installing Homebrew](https://docs.brew.sh/Installation) and then execute the following to install tools:
+
+```sh
+brew install bash gnu-tar gsed
+```
+
+### Configure your shell
+Modify your login shell's `PATH` to prefer the new tools over those in the existing environment. This example should work either with `zsh` (in $HOME/.zshrc) or `bash` (in $HOME/.bashrc):
```

Comment:
Is this something the user puts into their bash/zshrc or is it a command they run in the terminal?

Reply:
done in e379ecae